setup.bash

```diff
@@ -195,10 +195,19 @@ EOF
 
       printf "All done.\n"
       printf "Your %s branch has been reset to an initial commit.\n" "$primary_branch"
-      printf "Push to origin/%s with \`git push --force-with-lease\`\n" "$primary_branch"
 
-      printf "Review these TODO items:\n"
-      git grep -P -n -C 3 "TODO"
+      ok="${9:-$(ask_for "Push to origin? (yes/no)")}"
+      if [ "yes" != "$ok" ]; then
+        printf "Local repo not pushed.\n"
+        printf "Manually push to origin/%s with \`git push --force-with-lease\`\n" "$primary_branch"
+        printf "Then, find TODO items with \`git grep -n TODO\` and resolve them.\n"
```

Comment:
I do like this approach (git using a pager by default is very annoying) but would `-C 3` still be useful here for a bit of context? No idea why `-P` was ever the case though :P

Reply:
I think we should omit the context lines because:
- the TODO items are mostly independent of each other
- to resolve the TODO items, the developer would have to open the file(s) anyway
- having the line numbers is enough for the developer to open a file and quickly jump to that line, and then get the context
- adding 3+3 lines of context (+ 1 line for separator) fills up the screen pretty quickly, and this makes it more difficult to see all the TODO items as a list